status-im/status-react
# Show links inside activity-center mention

fixes #17353 

### Summary

Links weren't visible inside the `activity-center` inside a `mention` notification.



#### Platforms


- Android
- iOS

#### Areas that maybe impacted


##### Functional

- activity center

### Steps to test


- Open Status app with user A
- Open Status app with user B
- Both users should be in the same community/channel
- User B sends a message which mentions user A, followed by a url
- User A opens the `activity-center`
- User A sees the mention notification together with the url



### Before and after screenshots comparison

| Figma (if available) | iOS (if available)    | Android (if available)
| --- | --- | --- |
| Please embed Image/Video here of the before and after.  | <img src="https://github.com/status-im/status-mobile/assets/21865759/d8ef4b41-5c3a-4abf-ab61-92ea3517dc77" width=250 />  | Please embed Image/Video here of the before and after. |

status: ready
